### ISSUE TITLE
Wrong system recognation

### ISSUE THREAD

**Issue by DexterFromLab** [alvistack/ansible-collection-kubernetes#7]
Hello,

If I will configure ansible server as Ubuntu 20.04 and working node and k8s_master with RHEL 7.6 all servers operating system will be recognised as ubuntu server, and deployment script will throw error during apt get update, because RHEL not operate on this package manager.

I did workaround by using same OS everywhere.

**Comment by hswong3i** [comment]
Dear, it don’t suppose to be working by design…

For my clients and my daily use cases experience, I never face a mixed OS environment for deploying Kubernetes… Either Ubuntu/RHEL/OpenSUSE solely, but not mixed…

Anyway what I could only predicted cases is: mixing Linux and Windows node together, for running those Windows-only-based containers… BTW, even Azure AKS is based on Ubuntu 18.04… Moreover I don’t have any plan to support my Ansible roles/collection for Windows… So this mixed case should never happened…

Long story short: please don’t use mixed OS setup. You could check those GitLab CI result to see which OS is supposed to passing the installation (which Ubuntu 20.04/18.04 is primarily supported). In case you are using those OS(s) which currently failing in CI, please feel free to report issue or PR ;-)

**Comment by DexterFromLab** [comment]
So in my opinion is not a bug, is a feature :), I think this little fact can be mentioned in README and we can close a discussion ;)
